tests/nataliia_romanchuk/homework_5/test_dicts.py

```diff
@@ -0,0 +1,121 @@
+# DICTS
+import pytest
+
+some_dict = {"a": 123, "b": 456, "c": 789}
+some_dict_2 = {"c": 789, "a": 123, "b": 456}
+some_dict_3 = dict.fromkeys("abc")
+wierd_dict = {"a": 1, 2: "b",
+              None: "abc",
+              (1, "a"): "some",
+              True: "kek",
+              frozenset((1, 2)): "lol"}
+
+
+# is container
+def test_dict_comparison():
+    assert some_dict == some_dict_2
+    assert "a" in some_dict
+    assert len(some_dict) == 3
+    assert list(some_dict_2) == ['c', 'a', 'b']
+
+
+def test_dict_methods():
+    assert some_dict["a"] == 123
+    assert some_dict.get("a") == 123
+    assert some_dict["a"] == some_dict.get("a")
+    assert some_dict.get("invalid") == None
+    assert some_dict.get("invalid", "default_val") == 'default_val'
+
+
+@pytest.mark.parametrize("some", [{"a": 1, "b": 2}])
```

Comment:
no need in parametrize if there is only 1 case

Reply:
fixeed